res/css/views/right_panel/_UserInfo.scss

```diff
@@ -77,12 +76,27 @@ limitations under the License.
         that's why we had to put the margin to center on a parent div),
```

Comment:
is the padding + comment + max-width above still needed?

Reply:
Yes, I left the bare CSS required to make it work, underlying its doing the same trick as before but + position:relative with an absolutely positioned child which pins itself to fill the parent.